<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageProcessClass</class>
 <widget class="QMainWindow" name="ImageProcessClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1267</width>
    <height>967</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ImageProcess</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="ImageShow">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>600</width>
      <height>600</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>1221</width>
      <height>36</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="OpenImage">
       <property name="text">
        <string>打开图像</string>
       </property>
       <property name="iconSize">
        <size>
         <width>21</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="SaveImage">
       <property name="text">
        <string>保存图像</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ImageRecover">
       <property name="text">
        <string>处理图片</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="OpenStl">
       <property name="text">
        <string>打开三维图</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="OpenCloudSTL">
       <property name="text">
        <string>打开点云图-STL</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ThreeDRecover">
       <property name="text">
        <string>报警</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>670</y>
      <width>601</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="Message">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>581</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="SQ_QOpenGLWidget" name="OpenGLWidget">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>60</y>
      <width>600</width>
      <height>560</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>730</y>
      <width>601</width>
      <height>156</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0">
      <widget class="QLineEdit" name="ImageInfo"/>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="SaveCloudData">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="ReadCloudData">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="KeyPoint">
       <property name="text">
        <string>KeyPoint</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="OpenCloudPCD">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="KeyPointNumber"/>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="GetDepthImage2D">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1267</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="About"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>数据</string>
    </property>
    <addaction name="Database"/>
   </widget>
   <addaction name="menu_2"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action22">
   <property name="text">
    <string>22</string>
   </property>
  </action>
  <action name="About">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="Database">
   <property name="text">
    <string>Database</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>SQ_QOpenGLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header location="global">sq_qopenglwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <slots>
  <slot>SaveImageOriginalCv()</slot>
  <slot>aaaaaaaaaaaaa()</slot>
 </slots>
</ui>
